examples/run_language_modeling.py

```diff
@@ -75,6 +77,12 @@ class ModelArguments:
     cache_dir: Optional[str] = field(
         default=None, metadata={"help": "Where do you want to store the pretrained models downloaded from s3"}
     )
+    force_pad_token: bool = field(
+        default=False,
+        metadata={
+            "help": "Whether to force the addition of a padding token to tokenizer that does not already have one."
+        },
+    )
```

Comment:
I'm not a fan of this option personally (also see https://github.com/huggingface/transformers/issues/4122#issuecomment-644174575)

I'd rather the example scripts do not modify the specified tokenizer – I feel like advanced users should modify their tokenizer off-script.

Reply:
what do you think @patrickvonplaten?

Reply:
Hmm, same from my side. I think one should use `Trainer` with `DataCollatorForLanguageModeling` and use a fitting tokenizer.

GPT2 is a heavily used model though and it would be nice to allow to use it via this script. Another possibility I could see here is to add a `pad_token_id` to the args that would set PAD_TOKEN to the provided id. So for GPT2, one could do `--pad_token_id 50256`. On the other hand `pad_token_id` seems to be quite a specific param to add to the args, so it might be cleaner to just not allow this case and force the user to use `Trainer` + own `Datacollator`